lib/ruby-push-notifications.rb

```diff
@@ -1,3 +1,4 @@
 require 'ruby-push-notifications/version'
 require 'ruby-push-notifications/apns'
 require 'ruby-push-notifications/gcm'
+require 'ruby-push-notifications/mpns'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.